resources/content/db/mysql/create_db_and_user_dev.sql

```diff
@@ -1,8 +1,6 @@
-CREATE DATABASE cattle COLLATE = 'utf8_general_ci' CHARACTER SET = 'utf8';
+CREATE DATABASE IF NOT EXISTS cattle COLLATE = 'utf8_general_ci' CHARACTER SET = 'utf8';
```

Comment:
I was using it for personal reasons to stop sql from erroring when the script is run if there is a database and users already made. I will take it out and submit a separate  pr with the change and an explanation as to why.